Makefile

```diff
@@ -25,7 +25,8 @@ fix:
 	black sematic
 
 refresh-dependencies:
-	pip-compile --allow-unsafe requirements/requirements.in
+	bazel run //requirements:requirements38.update
```

Comment:
It turns out that some dependencies (including a transitive dep of Ray) require different packages for py 3.8 vs py 3.9 versions of their wheels. So we have to bifurcate our requirements.txt file.